[2021-12-29 09:26:13,077] {taskinstance.py:1035} INFO - Dependencies all met for <TaskInstance: scraping_daft_donegal.scrap_data scheduled__2021-12-28T09:25:00+00:00 [queued]>
[2021-12-29 09:26:14,630] {taskinstance.py:1035} INFO - Dependencies all met for <TaskInstance: scraping_daft_donegal.scrap_data scheduled__2021-12-28T09:25:00+00:00 [queued]>
[2021-12-29 09:26:14,658] {taskinstance.py:1241} INFO - 
--------------------------------------------------------------------------------
[2021-12-29 09:26:14,669] {taskinstance.py:1242} INFO - Starting attempt 1 of 4
[2021-12-29 09:26:14,700] {taskinstance.py:1243} INFO - 
--------------------------------------------------------------------------------
[2021-12-29 09:26:15,376] {taskinstance.py:1262} INFO - Executing <Task(SparkSubmitOperator): scrap_data> on 2021-12-28 09:25:00+00:00
[2021-12-29 09:26:15,775] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'scraping_daft_donegal', 'scrap_data', 'scheduled__2021-12-28T09:25:00+00:00', '--job-id', '5305', '--raw', '--subdir', 'DAGS_FOLDER/scraping_daft_donegal.py', '--cfg-path', '/tmp/tmpp80opz7c', '--error-file', '/tmp/tmpz9lixuz3']
[2021-12-29 09:26:15,844] {standard_task_runner.py:77} INFO - Job 5305: Subtask scrap_data
[2021-12-29 09:26:15,640] {standard_task_runner.py:52} INFO - Started process 23906 to run task
[2021-12-29 09:26:20,943] {logging_mixin.py:109} INFO - Running <TaskInstance: scraping_daft_donegal.scrap_data scheduled__2021-12-28T09:25:00+00:00 [running]> on host 6bdda4c19a85
[2021-12-29 09:26:26,808] {taskinstance.py:1429} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=clement.liscoet@gmail.com
AIRFLOW_CTX_DAG_OWNER=clemoni
AIRFLOW_CTX_DAG_ID=scraping_daft_donegal
AIRFLOW_CTX_TASK_ID=scrap_data
AIRFLOW_CTX_EXECUTION_DATE=2021-12-28T09:25:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-12-28T09:25:00+00:00
[2021-12-29 09:26:28,321] {base.py:79} INFO - Using connection to: id: spark_default. Host: spark://spark:7077, Port: None, Schema: , Login: ***, Password: ***, extra: {}
[2021-12-29 09:26:28,609] {spark_submit.py:360} INFO - Spark-Submit cmd: spark-submit --master spark://spark:7077 --conf spark.master=spark://spark:7077 --name scrap data --verbose /opt/***/jobs/get_ads_per_county.py donegal
[2021-12-29 09:28:32,393] {spark_submit.py:514} INFO - Using properties file: null
[2021-12-29 09:28:47,447] {spark_submit.py:514} INFO - WARNING: An illegal reflective access operation has occurred
[2021-12-29 09:28:47,460] {spark_submit.py:514} INFO - WARNING: Illegal reflective access by org.apache.spark.unsafe.Platform (file:/home/***/.local/lib/python3.6/site-packages/pyspark/jars/spark-unsafe_2.12-3.2.0.jar) to constructor java.nio.DirectByteBuffer(long,int)
[2021-12-29 09:28:47,473] {spark_submit.py:514} INFO - WARNING: Please consider reporting this to the maintainers of org.apache.spark.unsafe.Platform
[2021-12-29 09:28:47,477] {spark_submit.py:514} INFO - WARNING: Use --illegal-access=warn to enable warnings of further illegal reflective access operations
[2021-12-29 09:28:47,490] {spark_submit.py:514} INFO - WARNING: All illegal access operations will be denied in a future release
[2021-12-29 09:28:52,953] {spark_submit.py:514} INFO - Parsed arguments:
[2021-12-29 09:28:52,965] {spark_submit.py:514} INFO - master                  spark://spark:7077
[2021-12-29 09:28:52,985] {spark_submit.py:514} INFO - deployMode              null
[2021-12-29 09:28:52,997] {spark_submit.py:514} INFO - executorMemory          null
[2021-12-29 09:28:53,011] {spark_submit.py:514} INFO - executorCores           null
[2021-12-29 09:28:53,040] {spark_submit.py:514} INFO - totalExecutorCores      null
[2021-12-29 09:28:53,052] {spark_submit.py:514} INFO - propertiesFile          null
[2021-12-29 09:28:53,066] {spark_submit.py:514} INFO - driverMemory            null
[2021-12-29 09:28:53,076] {spark_submit.py:514} INFO - driverCores             null
[2021-12-29 09:28:53,086] {spark_submit.py:514} INFO - driverExtraClassPath    null
[2021-12-29 09:28:53,093] {spark_submit.py:514} INFO - driverExtraLibraryPath  null
[2021-12-29 09:28:53,110] {spark_submit.py:514} INFO - driverExtraJavaOptions  null
[2021-12-29 09:28:53,126] {spark_submit.py:514} INFO - supervise               false
[2021-12-29 09:28:53,134] {spark_submit.py:514} INFO - queue                   null
[2021-12-29 09:28:53,141] {spark_submit.py:514} INFO - numExecutors            null
[2021-12-29 09:28:53,150] {spark_submit.py:514} INFO - files                   null
[2021-12-29 09:28:53,167] {spark_submit.py:514} INFO - pyFiles                 null
[2021-12-29 09:28:53,181] {spark_submit.py:514} INFO - archives                null
[2021-12-29 09:28:53,200] {spark_submit.py:514} INFO - mainClass               null
[2021-12-29 09:28:53,213] {spark_submit.py:514} INFO - primaryResource         file:/opt/***/jobs/get_ads_per_county.py
[2021-12-29 09:28:53,235] {spark_submit.py:514} INFO - name                    scrap data
[2021-12-29 09:28:53,245] {spark_submit.py:514} INFO - childArgs               [donegal]
[2021-12-29 09:28:53,258] {spark_submit.py:514} INFO - jars                    null
[2021-12-29 09:28:53,272] {spark_submit.py:514} INFO - packages                null
[2021-12-29 09:28:53,281] {spark_submit.py:514} INFO - packagesExclusions      null
[2021-12-29 09:28:53,292] {spark_submit.py:514} INFO - repositories            null
[2021-12-29 09:28:53,301] {spark_submit.py:514} INFO - verbose                 true
[2021-12-29 09:28:53,317] {spark_submit.py:514} INFO - 
[2021-12-29 09:28:53,333] {spark_submit.py:514} INFO - Spark properties used, including those specified through
[2021-12-29 09:28:53,345] {spark_submit.py:514} INFO - --conf and those from the properties file null:
[2021-12-29 09:28:53,353] {spark_submit.py:514} INFO - (spark.master,spark://spark:7077)
[2021-12-29 09:28:53,362] {spark_submit.py:514} INFO - 
[2021-12-29 09:28:53,372] {spark_submit.py:514} INFO - 
[2021-12-29 09:55:40,663] {spark_submit.py:514} INFO - Main class:
[2021-12-29 09:55:40,706] {spark_submit.py:514} INFO - org.apache.spark.deploy.PythonRunner
[2021-12-29 09:55:40,711] {spark_submit.py:514} INFO - Arguments:
[2021-12-29 09:55:40,721] {spark_submit.py:514} INFO - file:/opt/***/jobs/get_ads_per_county.py
[2021-12-29 09:55:40,732] {spark_submit.py:514} INFO - null
[2021-12-29 09:55:40,737] {spark_submit.py:514} INFO - donegal
[2021-12-29 09:55:40,740] {spark_submit.py:514} INFO - --verbose
[2021-12-29 09:55:41,023] {spark_submit.py:514} INFO - Spark config:
[2021-12-29 09:55:41,025] {spark_submit.py:514} INFO - (spark.app.name,scrap data)
[2021-12-29 09:55:41,028] {spark_submit.py:514} INFO - (spark.master,spark://spark:7077)
[2021-12-29 09:55:41,033] {spark_submit.py:514} INFO - (spark.submit.pyFiles,)
[2021-12-29 09:55:41,039] {spark_submit.py:514} INFO - (spark.submit.deployMode,client)
[2021-12-29 09:55:41,044] {spark_submit.py:514} INFO - Classpath elements:
[2021-12-29 09:55:41,048] {spark_submit.py:514} INFO - 
[2021-12-29 09:55:41,050] {spark_submit.py:514} INFO - 
[2021-12-29 09:55:41,052] {spark_submit.py:514} INFO - 
[2021-12-29 11:40:06,152] {local_task_job.py:142} ERROR - Heartbeat time limit exceeded!
[2021-12-29 11:40:10,053] {process_utils.py:100} INFO - Sending Signals.SIGTERM to GPID 23906
[2021-12-29 11:53:00,966] {taskinstance.py:1411} ERROR - Received SIGTERM. Terminating subprocesses.
[2021-12-29 11:53:01,567] {spark_submit.py:644} INFO - Sending kill signal to spark-submit
[2021-12-29 11:53:02,791] {process_utils.py:66} INFO - Process psutil.Process(pid=24174, status='terminated', started='09:29:54') (24174) terminated with exit code None
[2021-12-29 11:53:13,910] {taskinstance.py:1703} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/taskinstance.py", line 1332, in _run_raw_task
    self._execute_task_with_callbacks(context)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/taskinstance.py", line 1458, in _execute_task_with_callbacks
    result = self._execute_task(context, self.task)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/taskinstance.py", line 1514, in _execute_task
    result = execute_callable(context=context)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/providers/apache/spark/operators/spark_submit.py", line 179, in execute
    self._hook.submit(self._application)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/providers/apache/spark/hooks/spark_submit.py", line 433, in submit
    self._process_spark_submit_log(iter(self._submit_sp.stdout))  # type: ignore
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/providers/apache/spark/hooks/spark_submit.py", line 482, in _process_spark_submit_log
    for line in itr:
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/taskinstance.py", line 1413, in signal_handler
    raise AirflowException("Task received SIGTERM signal")
airflow.exceptions.AirflowException: Task received SIGTERM signal
[2021-12-29 11:53:18,386] {taskinstance.py:1280} INFO - Marking task as UP_FOR_RETRY. dag_id=scraping_daft_donegal, task_id=scrap_data, execution_date=20211228T092500, start_date=20211229T092613, end_date=20211229T115317
[2021-12-29 11:53:24,353] {standard_task_runner.py:91} ERROR - Failed to execute job 5305 for task scrap_data
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/task/task_runner/standard_task_runner.py", line 85, in _start_by_fork
    args.func(args, dag=self.dag)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/cli/cli_parser.py", line 48, in command
    return func(*args, **kwargs)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/utils/cli.py", line 92, in wrapper
    return f(*args, **kwargs)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/cli/commands/task_command.py", line 292, in task_run
    _run_task_by_selected_method(args, dag, ti)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/cli/commands/task_command.py", line 107, in _run_task_by_selected_method
    _run_raw_task(args, ti)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/cli/commands/task_command.py", line 184, in _run_raw_task
    error_file=args.error_file,
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/utils/session.py", line 70, in wrapper
    return func(*args, session=session, **kwargs)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/taskinstance.py", line 1332, in _run_raw_task
    self._execute_task_with_callbacks(context)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/taskinstance.py", line 1458, in _execute_task_with_callbacks
    result = self._execute_task(context, self.task)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/taskinstance.py", line 1514, in _execute_task
    result = execute_callable(context=context)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/providers/apache/spark/operators/spark_submit.py", line 179, in execute
    self._hook.submit(self._application)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/providers/apache/spark/hooks/spark_submit.py", line 433, in submit
    self._process_spark_submit_log(iter(self._submit_sp.stdout))  # type: ignore
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/providers/apache/spark/hooks/spark_submit.py", line 482, in _process_spark_submit_log
    for line in itr:
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/taskinstance.py", line 1413, in signal_handler
    raise AirflowException("Task received SIGTERM signal")
airflow.exceptions.AirflowException: Task received SIGTERM signal
[2021-12-29 11:53:33,111] {process_utils.py:66} INFO - Process psutil.Process(pid=23928, status='terminated', started='09:26:28') (23928) terminated with exit code None
[2021-12-29 11:53:33,736] {process_utils.py:66} INFO - Process psutil.Process(pid=23906, status='terminated', exitcode=1, started='09:26:15') (23906) terminated with exit code 1
